conf.yaml

```diff
@@ -150,6 +156,15 @@ repos:
 contents_title:     Elastic Stack and Product Documentation
 
 contents:
+    -   title:      "Elastic Stack"
+        sections:
+          -
+            title:      Installation and Upgrade Guide
+            prefix:     en/elasticstack
```

Comment:
Should we do `elastic-stack` or `elasticstack`? I misread it at first as `elasticsearch` without the dash.